addons/website/static/src/components/dialog/edit_menu.xml

```diff
@@ -16,6 +16,7 @@
             </label>
             <div class="col-md-9">
                 <input id="url_input" t-ref="url-input" type="text" t-model="url.input.value" class="form-control" t-att-class="{ 'is-invalid': url.input.hasError }"/>
+                <small class="form-text text-muted">Hint: Type '/' to search an existing page and '#' to link to an anchor.</small>
```

Comment:
I'll move this to another PR and keep this PR open in draft, we'll see what to do with it when you are back

Reply:
https://github.com/odoo/odoo/pull/97733 -> also removed the text-muted class which had no effect